### PR TITLE
Lint all

### PR DESCRIPTION
**Pull request by orenco-resonai**

### CODE REVIEW

File: sample-apps/communication/main.js [hunk 1]
```diff
@@ -18,9 +18,9 @@ async function init () {
     if (data.action === 'navigationSuccess') {
       // Briefly show the navigation success message
       const msgElement = document.getElementById('dest-reached')
-      msgElement.style.display = "block"
-      setTimeout(function() {
-        msgElement.style.display = "none"
+      msgElement.style.display = 'block'
+      setTimeout(function () {
```
Comment (yoav-resonai) [inline sample-apps/communication/main.js:22]: use anonymous function:
`() => { msgElement.style.display = 'none' }`

File: package.json [hunk 2]
```diff
@@ -10,7 +10,6 @@
     "deploy": "vite build && git push origin --delete gh-pages && git subtree push --prefix dist origin gh-pages"
   },
   "dependencies": {
-    "//@resonai/vera-sdk": "^0.0.11",
     "@resonai/vera-sdk": "file:../resonai2/irocket/client/web/verasdk/"
```
Comment (yoav-resonai) [inline package.json:13]: don't we need the global? or at least add a comment

Comment (orenco-resonai) [reply package.json:13]: Reverting this in a later PR

File: sample-apps/communication/main.js (hunk 1 above)
Comment (orenco-resonai) [reply sample-apps/communication/main.js:22]: Done

**Review by yoav-resonai — APPROVED** [review]

**Pull request merged**